docs/guides/python_chip_controller_building.md

```diff
@@ -298,6 +319,8 @@ chip-device-ctrl > ble-adapter-print
 
 ### `ble-debug-log`
 
+> BLE adapter operations is not yet supported in REPL
```

Comment:
Can you file an issue tracking these BLE operations and getting those implemented in the REPL?